org-formation/_scp.yaml

```diff
@@ -175,3 +175,20 @@
             Action:
               - organizations:LeaveOrganization
             Resource: "*"
+
+  RequireSecureS3:
+    Type: OC::ORG::ServiceControlPolicy
+    Properties:
+      PolicyName: RequireSecureS3
+      Description: Restrict users from accessing S3 over HTTP
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DenyInsecureS3
+            Effect: Deny
+            Action:
+              - 's3:*'
+            Resource: '*'
+            Condition:
+              Bool:
+                'aws:SecureTransport': false
```

Comment:
The following AWS KB seems to indicate that we would need `"Principal": "*"` in this PolicyDocument?
https://repost.aws/knowledge-center/s3-bucket-policy-for-config-rule

You should test to see what type of message you get when attempting to get files over http

Reply:
Thanks, I saw that KB article but it's describing a bucket policy and this PR is creating a service control policy, so I don't think the `Principal` is needed, I don't see it in any of the examples [here](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps_examples_general.html)

I'm working on setting up a test in the sandbox organization, I'll comment my results in the ticket.

Reply:
ohh, hmm.  i'm not sure `'aws:SecureTransport': false` is supported for SCP but i guess the easiest thing to do is try a deploy and see what happens.

Reply:
Oh, hrm. I tried this out in the test org, and it created the SCP just fine but it had no effect. We'll have to go through each bucket individually and either add the policy or suppress the finding.